scripts/route-file-content.js

```diff
@@ -58,25 +60,30 @@ function routeFileContent(outputPath, callback) {
       );
     }
 
-    var index = React.createFactory(require('../pages/index.jsx'));
```

Comment:
Most of this was changed because their API has changed, but I basically copied the render bits from advocacy.mozilla.org, because it was also cleaner.